Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -374,12 +376,28 @@ protected void onVisibilityChanged(@NonNull View changedView, int visibility) {
                                 } else if (VUNGLE_MREC == adConfig.getAdSize()) {
                                     vungleMrecAd = sVungleRouter.getVungleMrecAd(placementReferenceId, adConfig);
                                     if (vungleMrecAd != null) {
-                                        View adView = vungleMrecAd.renderNativeView();
+                                        final View adView = vungleMrecAd.renderNativeView();
                                         if (adView != null) {
                                             isLoadSuccess = true;
-                                            // Honoring the server dimensions forces the WebView to be the size of the MREC
-                                            AdViewController.setShouldHonorServerDimensions(layout);
-                                            layout.addView(adView);
+                                            final int width = ViewUtility.dpToPixels(mContext, VUNGLE_MREC.getWidth());
+                                            final int height = ViewUtility.dpToPixels(mContext, VUNGLE_MREC.getHeight());
+                                            RelativeLayout mrecViewWrapper = new RelativeLayout(mContext) {
+                                                @Override
+                                                protected void onAttachedToWindow() {
```

Comment:
isn't layout.addView(mrecViewWrapper, new LayoutParams(...)); enought?

Reply:
It can't work by this way.

Reply:
I meant something like:
               ```
  //                                        Honoring the server dimensions forces the WebView to be the size of the MREC
                                            AdViewController.setShouldHonorServerDimensions(layout);
                                            final int width = ViewUtility.dpToPixels(mContext, VUNGLE_MREC.getWidth());
                                            final int height = ViewUtility.dpToPixels(mContext, VUNGLE_MREC.getHeight());
                                            RelativeLayout mrecViewWrapper = new RelativeLayout(mContext);
                                            mrecViewWrapper.addView(adView, new RelativeLayout.LayoutParams(width, height));
                                            layout.addView(mrecViewWrapper);
```

Reply:
Sure, don't work. mrecViewWrapper will stretch to parent.

Reply:
sorry, I was proposing to set params to the outer container:
layout.addView(mrecViewWrapper, new RelativeLayout.LayoutParams(width, height));

Reply:
fixed.